docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -36,8 +36,8 @@ Let us walk through the changes that are required in different phases of your ne
 A Fabric distributed application's business logic code spans two layers as illustrated in the network model:
```

Comment:
Can you revert all changes to this file? I've made extensive changes here, including the lines you have modified below (making fixes similar to what you were attempting). That way we won't run into conflicts.